admyral/config/config.py

```diff
@@ -173,13 +154,34 @@ class GlobalConfig(BaseModel):
     posthog_api_key: str = ADMYRAL_POSTHOG_API_KEY
     posthog_host: str = ADMYRAL_POSTHOG_HOST
     environment: str = ADMYRAL_ENV
-    posthog_disabled: bool = read_posthog_disabled()
+    telemetry_disabled: bool = False
```

Comment:
We also don't need a default value here because we will always pass down a value.